acceptance/cmd/sync/from-file/script

```diff
@@ -0,0 +1,21 @@
+mkdir "project-folder" "project-folder/blob" "ignored-folder" "ignored-folder/folder1" ".git"
+touch "project-folder/app.yaml" "project-folder/app.py" "project-folder/query.sql" "project-folder/blob/1" "project-folder/blob/2"
+touch "ignored-folder/script.py" "ignored-folder/folder1/script.py" "ignored-folder/folder1/script.yaml" "ignored-folder/folder1/big-blob"
+cat > .gitignore << EOF
+ignored-folder/
+script
+output.txt
+repls.json
+EOF
+cat > .databricksignore << EOF
+project-folder/blob/*
+project-folder/*.sql
```

Comment:
You could check different version of .databricksignore:

- refers to exact file (existing / missing(
- refers to exact directory (existing / missing)
- contains "**"

Reply:
added in the follow-up commit